data/remnant/remnant ships.txt

```diff
@@ -892,48 +892,34 @@ ship "Merganser"
 		category "Interceptor"
 		licenses
 			Remnant
-		"cost" 6830000
-		"shields" 3000
-		"hull" 5000
+		"cost" 4830000
+		"shields" 3500
+		"hull" 3000
 		"required crew" 1
 		"bunks" 3
 		"cargo space" 11
 		"fuel capacity" 600
-		"ramscoop" 1.75
-		"mass" 30
-		"drag" 1.5
+		"ramscoop" 1.25
+		"mass" 90
+		"drag" 1.3
 		"heat dissipation" .9
 		"gaslining" 1
-		"outfit space" 219
-		"shield generation" 2.8
+		"outfit space" 180
+		"weapon capacity" 60
+		"engine capacity" 69
+		"shield generation" 1.4
 		"shield energy" 1.2
-		"hull repair rate" 1.8
-		"hull energy" 0.6
-		"cloak" .04
-		"cloaking energy" 2
-		"cloaking fuel" .05
+		"hull repair rate" 0.6
+		"hull energy" 0.4
+		"cloak" .03
+		"cloaking energy" 4
+		"cloaking fuel" .075
 		"atmosphere scan" 100
 		"outfit scan power" 12
 		"outfit scan efficiency" 12
 		"tactical scan power" 26
-		"ion resistance" 0.1
-		"ion resistance energy" -0.05
-		"scramble resistance" 0.1
-		"scramble resistance energy" -0.05
-		"thrust" 5.0
-		"turn" 60.0
-		"reverse thrust" 5.0
-		"flare sprite" "effect/remnant flare/small"
-			"frame rate" 5
-		"flare sound" "plasma tiny"
-		"steering flare sprite" "effect/remnant flare/small"
-			"frame rate" 5
-		"steering flare sound" "plasma tiny"
-		"reverse flare sprite" "effect/remnant flare/small"
-			"frame rate" 5
-		"reverse flare sound" "plasma tiny"
-		"weapon capacity" 85
-		"engine capacity" 70
+		"ion resistance" 0.025
+		"scramble resistance" 0.025
```

Comment:
that thrust is free of heat and energy tho, we shouldnt really have that

Reply:
That's a fair comment; although 5 reverse thrust is about less than 1% of the reverse of a anvil engine. Calculating it out, the appropriate reverse heat should be 0.012, unless I missed something. (It's been a while, so please, someone else double check that)

Reply:
5 reverse thrust is about 50% the reverse of an anvil engine, because these numbers are not the ones you see in game. An anvil engine has 10.3 reverse thrust.and 1.45 heat, so this should have about 0.7 reverse heat.